src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+    private final Color[] colorSet = Color.values();
```

Comment:
Please choose better name for this variable, because, it's not a Set, it's an array :) You will learn about `Set` in the nearest future.